package.json

```diff
@@ -42,7 +42,7 @@
   "main": "./dist/index.js",
   "name": "eslint-plugin-sql",
   "peerDependencies": {
-    "eslint": ">=7.9.0"
+    "eslint": ">=6.8.0"
```

Comment:
This should not be changed manually.

https://github.com/gajus/eslint-plugin-sql/blob/2b449059dd6e3d11609ad7f6020c4bac6e73ef8a/.travis.yml#L15

Reply:
Yeah, that's why I reverted the change, and still, CI is not passing.

Reply:
I would try adding `--legacy-peer-deps`.

Reply:
Done, Working!